config/params/loader_test.go

```diff
@@ -194,6 +194,9 @@ func TestLoadConfigFile(t *testing.T) {
 		mnf, err := params.UnmarshalConfigFile(mainnetConfigFile, nil)
 		require.NoError(t, err)
 		fields := fieldsFromYamls(t, append(mainnetPresetsFiles, mainnetConfigFile))
+		// TODO(#14714): remove hard coded sizes once spec is updated
+		mnf.MaxChunkSize = 15728640
+		mnf.GossipMaxSize = 15728640
```

Comment:
These feel like a temporary spec violation. I'm not sure we should merge & release this until https://github.com/ethereum/consensus-specs/pull/4041 merges.